src/engine/cachingreader.cpp

```diff
@@ -218,6 +223,7 @@ void CachingReader::process() {
     ReaderStatusUpdate update;
     while (m_readerStatusUpdateFIFO.read(&update, 1) == 1) {
         auto pChunk = update.takeFromWorker();
+        qDebug() << "CachingReader::process()" << update.status;
```

Comment:
This debug log will be printed too frequently, not a good idea.

Reply:
Ups a leftover.